missions/2PzD_Template_v3_0.VR/customization/geardefs/US/gearDefUSMC.sqf

```diff
@@ -1,48 +1,21 @@
 //======================== US Marine Corps Uniform Definitions ========================
```

Comment:
Let's just delete the entire USMC loadout, GearDef, and Composition for now. We can't make pacific missions anymore anyway.